docs/source/faq.rst

```diff
@@ -70,3 +70,11 @@ users use high-level functions like
 HDFS name node, finds the locations of all of the blocks of data, and sends
 that information to the scheduler so that it can make smarter decisions and
 improve load times for users.
+
+
+PermissionError [Errno 13] Permission Denied: `/root/.dask`
```

Comment:
Also, in rst double backticks are used for code highlighting, not single backticks

    ``x + 1``

I also suspect that code highlighting simply doesn't work in titles.  I would stick to normal text.

If you want to build locally to see how things look (no expectation that you actually do this) here are the docs: http://dask.pydata.org/en/latest/develop.html#contributing-to-documentation